pkg/testing/integration/util.go

```diff
@@ -80,6 +80,7 @@ func getCmdBin(loc *string, bin, def string) (string, error) {
 			}
 		}
 	}
+	fmt.Printf("Using `%s` binary at `%s`\n", bin, *loc)
```

Comment:
Is this printf left in from debugging or intentional? I can see us adding this, since we keep changing what bin we are using during tests ...

Reply:
Oh left from debugging, but yeh might be fine to just leave it in